images/Dockerfile.ubuntu

```diff
@@ -34,20 +34,29 @@ FROM ${BASE_IMAGE} AS ubuntu-20.04-upstream
 
 FROM ${BASE_IMAGE} AS ubuntu-22.04-upstream
 
+###############################################################
+####                build nohang from source               ####
+###############################################################
+FROM ubuntu-22.04-upstream as nohang-builder
+WORKDIR /root
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  make fakeroot git ca-certificates
+
+RUN git clone -b master --depth=1 https://github.com/hakavlad/nohang.git \
+  && cd nohang \
+  && echo > deb/DEBIAN/postinst \
+  && deb/build.sh
```

Comment:
My feeling is that it would be better to have this as an independent image, or better yet, as a luet package https://github.com/kairos-io/packages/

Reply:
yes, packages seem better way to go. Honestly, this is more like a hack.

Reply:
I think I spoke too soon. I don't really know how nohang works but it seems to depend on the init system for installation. That means we would need to have an openrc and a systemd version of the package. Since no other distros are having the issue maybe we stick to a solution like yours in the meantime. Let me try a few things to see what works best